skylib/zip.bzl

```diff
@@ -50,6 +50,12 @@ def _gzip(ctx, artifact, out, decompress, options, mnemonic):
             ) + (options or []),
             inputs = [artifact],
             outputs = [out],
+            execution_requirements = {
```

Comment:
Should we add this same change to the `ctx.actions.run_shell` below?

Reply:
yeah I was on the fence about it since it's harder to be sure that a user's gzip is actually behaving according to our "CPU load not high enough to be economical".
But in practice they are probably all gzip-equivalent programs